web/components/navbar/index.js

```diff
@@ -79,35 +81,36 @@ class Navbar extends React.Component {
       case DISPLAY_TYPE.ONLY_LOGO:
         break;
       case DISPLAY_TYPE.LOGO_AND_BUTTON:
-        navbarRight = <div className="flex ai-center">{button}</div>;
+        navbarRight = <div className={LINK_CLASS}>{button}</div>;
         break;
       case DISPLAY_TYPE.LOGO_AND_LINKS:
         navbarRight = linkElements.map(l => (
           <div
             key={key += 1}
-            className="flex ai-center margin-sides-l">
+            className={`${LINK_CLASS} scale-hide-phablet`}>
             {l}
           </div>
         ));
+        navbarRight.push(<div key={key += 1} className={LINK_CLASS}>{button}</div>);
         break;
       default:
         navbarRight = linkElements.concat([button]).map(l => (
           <div
             key={key += 1}
-            className="flex ai-center margin-sides-l">
+            className={`${LINK_CLASS} scale-hide-phablet`}>
```

Comment:
Wouldn’t this hide the button if the viewport is phablet? We only want to hide the links here